README.md

```diff
@@ -31,14 +31,16 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the DOM?
-The dom stands for Document Object Model. This is 
+The dom stands for Document Object Model. This is a web interface and its a refrence to the actual code. 
```

Comment:
This is a web interface that allows elements on the Window object be manipulated via javascript. *